packages/core/src/Header/Header.tsx

```diff
@@ -36,21 +36,19 @@ export const HvHeader = (props: HvHeaderProps) => {
     ...others
   } = useDefaultProps("HvHeader", props);
 
-  const { classes, cx, css } = useClasses(classesProp);
+  const { classes, cx } = useClasses(classesProp);
 
   return (
     <header
+      style={
+        {
+          "--header-position": position,
+        } as React.CSSProperties
+      }
```

Comment:
I updated the PR with this 🙏 I just left the type assertion for the template file since we don't want to promote the utils package and this template is used in the CLI